sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -416,3 +416,7 @@ CREATE TABLE `pro_assessments` (
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB AUTO_INCREMENT=1;
 #EndIf
+
+#IfNotRow2D list_options list_id lists option_id LBF_Validations
```

Comment:
Hi @im-Amitto , This PR is testing nicely and bringing into codebase. Very nice work! I am going to make a change to the above line in a subsequent commit, which should be:
```
#IfNotRow2D list_options list_id LBF_Validations option_id future_date
```
Note this "function" will then only run the sql statement here if the there is not yet an entry in list_options with list_id LBF_Validations and option_id future_date

To see all the possible "functions" check out the top of this script.

Reply:
@im-Amitto ,
Here's the subsequent commit with code change. Also note that we increment $v_database in version.php whenever we make a database change in upgrade script. This is so the brave souls that run OpenEMR of the development codebase know when they need to run upgrade script on the database (using the admin.php script).
https://github.com/openemr/openemr/commit/0be0d4eae37366ce4cba69f16cb86917a35fd182